src/ui/callouts/widget_simplelinecallout.ui

```diff
@@ -203,39 +216,43 @@
        </property>
       </widget>
      </item>
-     <item row="0" column="0">
-      <widget class="QLabel" name="label">
-       <property name="text">
-        <string>Line style</string>
+     <item row="3" column="1">
+      <widget class="QgsDoubleSpinBox" name="mOffsetFromLabelSpin">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+         <horstretch>1</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
        </property>
-      </widget>
-     </item>
-     <item row="4" column="0" colspan="3">
-      <widget class="QCheckBox" name="mDrawToAllPartsCheck">
-       <property name="text">
-        <string>Draw lines to all feature parts</string>
+       <property name="decimals">
+        <number>6</number>
        </property>
-      </widget>
-     </item>
-     <item row="4" column="3">
-      <widget class="QgsPropertyOverrideButton" name="mDrawToAllPartsDDBtn">
-       <property name="text">
-        <string>…</string>
+       <property name="maximum">
+        <double>100000.000000000000000</double>
+       </property>
+       <property name="singleStep">
+        <double>0.200000000000000</double>
+       </property>
+       <property name="value">
+        <double>0.000000000000000</double>
+       </property>
+       <property name="showClearButton" stdset="0">
+        <bool>true</bool>
        </property>
       </widget>
      </item>
-     <item row="5" column="0">
-      <widget class="QLabel" name="mAnchorPointLbl">
+     <item row="6" column="0">
+      <widget class="QLabel" name="mAnchorPointLbl_2">
        <property name="text">
-        <string>Anchor point</string>
+        <string>Label anchor point</string>
        </property>
       </widget>
      </item>
-     <item row="5" column="1" colspan="2">
-      <widget class="QComboBox" name="mAnchorPointComboBox"></widget>
+     <item row="6" column="1" colspan="2">
+      <widget class="QComboBox" name="mLabelAnchorPointComboBox"/>
```

Comment:
Needs to be added in the <tabstops> section.

Reply:
Fixed